tests/src/test/scala/system/basic/WskActionTests.scala

```diff
@@ -267,16 +269,6 @@ class WskActionTests extends TestHelpers with WskTestHelpers with JsHelpers {
     }
   }
 
-  it should "reject an invoke with the wrong parameters set" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
I see why you moved this test to WskBasicUsageTests as it tests the functionality of the CLI. However, is there a similar test for just the REST interface?

Reply:
This test is testing the CLI, so it will be moved to the repo of cli to test CLI. We do not need to have a REST counterpart to test, because it is an cli only functionality as "invoke with the wrong parameters".

Reply:
Does there exist a test for testing the REST interface with bad parameters though?